src/libs/conduit/conduit_node.cpp

```diff
@@ -420,7 +420,8 @@ Node::mmap(const std::string &stream_path,
     m_mmaped = false;
 
     m_schema->set(schema);
-    walk_schema(this,m_schema,m_data);
+#warning "I don't know if this is ok(mmap). It might need to be set to the defautl alloc_id"
+    walk_schema(this,m_schema,m_data, m_allocator_id);
 
```

Comment:
mmap itself is like using a special allocator. When mmap is used -- it should dictate the the default alloc is used. 

the way mmap works now -- you setup it up, and if you deviate (set that isn't compat, etc) you are back to default

Reply:
flip flopped on this, it should propagate the alloc like set_external does